code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -305,17 +305,6 @@
 					target << "<span class='danger'>Your gaze is forcibly drawn into a blinding red light. You fall to the floor as conscious thought is wiped away.</span>"
 					target.Weaken(12)
 					sleep(20)
-					if(isloyal(target))
```

Comment:
While I'm somewhat negative on the change itself, as it stands this won't do what you want. What you want to do:
* Check if loyal
* If so, set shadowling's "enthralling" variable to 0
* Optionally, give an informative message to each party
* Then, return